### PR TITLE
Fixes Quarkus version and images dir in scripting guide

### PR DESCRIPTION
**Pull request by abelsromero**
I went throught the scripting guide and fixed a few things.

* Added attributes include. This in turn fixed the architecture image path and allowed to use 'quarkus-version' in code examples.
* Removed duplicated commands in jbang examples. Was it intentional?
* Some minnor style fixes: comas, typo, use auto-numbered lists.

Also found this problems:
* There's a mention to "hot reload", however it does not work with jbang afaik. Imo, should be removed.
* Swagger example fails. Swagger-ui loads but with message "Failed to load API definition."

### CODE REVIEW

File: docs/src/main/asciidoc/scripting.adoc [hunk 1]
```diff
@@ -231,7 +232,7 @@ We are using `curl -w "\n"` in this example to avoid your terminal printing a '%
 ====
 
 [TIP]
-.Why is `quarkus-resteasy` not resolved ? 
+.Why is `quarkus-resteasy` not resolved ?
```
Comment (gsmet) [inline docs/src/main/asciidoc/scripting.adoc:235]:
```suggestion
.Why is `quarkus-resteasy` not resolved?
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks! Can you adjust the other issue I spotted and squash everything?

File: docs/src/main/asciidoc/scripting.adoc (hunk 1 above)
Comment (abelsromero) [reply docs/src/main/asciidoc/scripting.adoc]: Done, there was in fact another ` ?` I also fixed.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
The changes looks good to me but I will let @maxandersen answer on the more fundamental issues you reported.

**Comment by maxandersen** [comment]
> * There's a mention to "hot reload", however it does not work with jbang afaik. Imo, should be removed.

agreed, please do. Just a miss when I altered the original rest example. fyi: It will be able at some point; just not yet.

**Comment by maxandersen** [comment]
> * Swagger example fails. Swagger-ui loads but with message "Failed to load API definition."

what error do you get exactly ? 

It works for me when I use the source below and visits http://0.0.0.0:8080/swagger-ui/

```java
//usr/bin/env jbang "$0" "$@" ; exit $?
//DEPS io.quarkus:quarkus-resteasy:1.8.0.Final
//DEPS io.quarkus:quarkus-smallrye-openapi:1.8.0.Final
//DEPS io.quarkus:quarkus-swagger-ui:1.8.0.Final
//Q:CONFIG quarkus.swagger-ui.always-include=true
//JAVA_OPTIONS -Djava.util.logging.manager=org.jboss.logmanager.LogManager
//JAVAC_OPTIONS -parameters

import io.quarkus.runtime.Quarkus;
import javax.enterprise.context.ApplicationScoped;
import javax.ws.rs.GET;
import javax.ws.rs.Path;

@Path("/hello")
@ApplicationScoped
public class quarkusrest {

    @GET
    public String sayHello() {
        return "Hello from Quarkus with jbang.dev";
    }

    public static void main(String[] args) {
        Quarkus.run(args);
    }
}
```

**Review by maxandersen — APPROVED** [review]
contents is fine - removing mention of hot load would be good. approving under that assumption.

**Comment by abelsromero** [comment]
"Hot reload" reference removed.

> It works for me when I use the source below and visits http://0.0.0.0:8080/swagger-ui/

You are right, my mistake. I did not add the openapi dep.
There's still this "INVALID" shown below, but that's not jbang related.

image

**Comment by gsmet** [comment]
Thanks Abel!

**Pull request merged**